2018-06-20 10:53:34,410:INFO: BiLSTM-CRF for Chinese NER task
2018-06-20 10:59:49,203:INFO: 2018-06-20 10:59:41 epoch 1, step 1, loss: 127.1, global_step: 1
2018-06-20 10:59:49,203:INFO: 2018-06-20 10:59:41 epoch 1, step 1, loss: 127.1, global_step: 1
2018-06-20 10:59:49,203:INFO: 2018-06-20 10:59:41 epoch 1, step 1, loss: 127.1, global_step: 1
2018-06-20 11:01:39,562:INFO: 2018-06-20 10:59:41 epoch 1, step 30, loss: 44.92, global_step: 30
2018-06-20 11:01:39,562:INFO: 2018-06-20 10:59:41 epoch 1, step 30, loss: 44.92, global_step: 30
2018-06-20 11:01:39,562:INFO: 2018-06-20 10:59:41 epoch 1, step 30, loss: 44.92, global_step: 30
2018-06-20 11:01:40,273:INFO: ===========validation / test===========
2018-06-20 11:01:40,273:INFO: ===========validation / test===========
2018-06-20 11:01:40,273:INFO: ===========validation / test===========
2018-06-20 11:01:59,831:INFO: 2018-06-20 11:01:53 epoch 2, step 1, loss: 51.96, global_step: 31
2018-06-20 11:01:59,831:INFO: 2018-06-20 11:01:53 epoch 2, step 1, loss: 51.96, global_step: 31
2018-06-20 11:01:59,831:INFO: 2018-06-20 11:01:53 epoch 2, step 1, loss: 51.96, global_step: 31
2018-06-20 11:03:42,652:INFO: 2018-06-20 11:01:53 epoch 2, step 30, loss: 19.23, global_step: 60
2018-06-20 11:03:42,652:INFO: 2018-06-20 11:01:53 epoch 2, step 30, loss: 19.23, global_step: 60
2018-06-20 11:03:42,652:INFO: 2018-06-20 11:01:53 epoch 2, step 30, loss: 19.23, global_step: 60
2018-06-20 11:03:43,184:INFO: ===========validation / test===========
2018-06-20 11:03:43,184:INFO: ===========validation / test===========
2018-06-20 11:03:43,184:INFO: ===========validation / test===========
2018-06-20 11:03:59,944:INFO: 2018-06-20 11:03:55 epoch 3, step 1, loss: 29.25, global_step: 61
2018-06-20 11:03:59,944:INFO: 2018-06-20 11:03:55 epoch 3, step 1, loss: 29.25, global_step: 61
2018-06-20 11:03:59,944:INFO: 2018-06-20 11:03:55 epoch 3, step 1, loss: 29.25, global_step: 61
2018-06-20 11:05:47,835:INFO: 2018-06-20 11:03:55 epoch 3, step 30, loss: 30.1, global_step: 90
2018-06-20 11:05:47,835:INFO: 2018-06-20 11:03:55 epoch 3, step 30, loss: 30.1, global_step: 90
2018-06-20 11:05:47,835:INFO: 2018-06-20 11:03:55 epoch 3, step 30, loss: 30.1, global_step: 90
2018-06-20 11:05:48,410:INFO: ===========validation / test===========
2018-06-20 11:05:48,410:INFO: ===========validation / test===========
2018-06-20 11:05:48,410:INFO: ===========validation / test===========
2018-06-20 11:06:03,085:INFO: 2018-06-20 11:06:01 epoch 4, step 1, loss: 21.67, global_step: 91
2018-06-20 11:06:03,085:INFO: 2018-06-20 11:06:01 epoch 4, step 1, loss: 21.67, global_step: 91
2018-06-20 11:06:03,085:INFO: 2018-06-20 11:06:01 epoch 4, step 1, loss: 21.67, global_step: 91
2018-06-20 11:07:48,586:INFO: 2018-06-20 11:06:01 epoch 4, step 30, loss: 30.54, global_step: 120
2018-06-20 11:07:48,586:INFO: 2018-06-20 11:06:01 epoch 4, step 30, loss: 30.54, global_step: 120
2018-06-20 11:07:48,586:INFO: 2018-06-20 11:06:01 epoch 4, step 30, loss: 30.54, global_step: 120
2018-06-20 11:07:49,096:INFO: ===========validation / test===========
2018-06-20 11:07:49,096:INFO: ===========validation / test===========
2018-06-20 11:07:49,096:INFO: ===========validation / test===========
2018-06-20 11:08:03,266:INFO: 2018-06-20 11:08:01 epoch 5, step 1, loss: 10.59, global_step: 121
2018-06-20 11:08:03,266:INFO: 2018-06-20 11:08:01 epoch 5, step 1, loss: 10.59, global_step: 121
2018-06-20 11:08:03,266:INFO: 2018-06-20 11:08:01 epoch 5, step 1, loss: 10.59, global_step: 121
2018-06-20 11:09:51,883:INFO: 2018-06-20 11:08:01 epoch 5, step 30, loss: 11.48, global_step: 150
2018-06-20 11:09:51,883:INFO: 2018-06-20 11:08:01 epoch 5, step 30, loss: 11.48, global_step: 150
2018-06-20 11:09:51,883:INFO: 2018-06-20 11:08:01 epoch 5, step 30, loss: 11.48, global_step: 150
2018-06-20 11:09:52,573:INFO: ===========validation / test===========
2018-06-20 11:09:52,573:INFO: ===========validation / test===========
2018-06-20 11:09:52,573:INFO: ===========validation / test===========
2018-06-20 11:10:09,003:INFO: 2018-06-20 11:10:05 epoch 6, step 1, loss: 12.34, global_step: 151
2018-06-20 11:10:09,003:INFO: 2018-06-20 11:10:05 epoch 6, step 1, loss: 12.34, global_step: 151
2018-06-20 11:10:09,003:INFO: 2018-06-20 11:10:05 epoch 6, step 1, loss: 12.34, global_step: 151
2018-06-20 11:11:57,443:INFO: 2018-06-20 11:10:05 epoch 6, step 30, loss: 4.91, global_step: 180
2018-06-20 11:11:57,443:INFO: 2018-06-20 11:10:05 epoch 6, step 30, loss: 4.91, global_step: 180
2018-06-20 11:11:57,443:INFO: 2018-06-20 11:10:05 epoch 6, step 30, loss: 4.91, global_step: 180
2018-06-20 11:11:58,058:INFO: ===========validation / test===========
2018-06-20 11:11:58,058:INFO: ===========validation / test===========
2018-06-20 11:11:58,058:INFO: ===========validation / test===========
2018-06-20 11:12:16,677:INFO: 2018-06-20 11:12:11 epoch 7, step 1, loss: 10.91, global_step: 181
2018-06-20 11:12:16,677:INFO: 2018-06-20 11:12:11 epoch 7, step 1, loss: 10.91, global_step: 181
2018-06-20 11:12:16,677:INFO: 2018-06-20 11:12:11 epoch 7, step 1, loss: 10.91, global_step: 181
2018-06-20 11:14:00,927:INFO: 2018-06-20 11:12:11 epoch 7, step 30, loss: 4.005, global_step: 210
2018-06-20 11:14:00,927:INFO: 2018-06-20 11:12:11 epoch 7, step 30, loss: 4.005, global_step: 210
2018-06-20 11:14:00,927:INFO: 2018-06-20 11:12:11 epoch 7, step 30, loss: 4.005, global_step: 210
2018-06-20 11:14:01,476:INFO: ===========validation / test===========
2018-06-20 11:14:01,476:INFO: ===========validation / test===========
2018-06-20 11:14:01,476:INFO: ===========validation / test===========
2018-06-20 11:14:16,500:INFO: 2018-06-20 11:14:14 epoch 8, step 1, loss: 8.602, global_step: 211
2018-06-20 11:14:16,500:INFO: 2018-06-20 11:14:14 epoch 8, step 1, loss: 8.602, global_step: 211
2018-06-20 11:14:16,500:INFO: 2018-06-20 11:14:14 epoch 8, step 1, loss: 8.602, global_step: 211
2018-06-20 11:16:04,314:INFO: 2018-06-20 11:14:14 epoch 8, step 30, loss: 7.724, global_step: 240
2018-06-20 11:16:04,314:INFO: 2018-06-20 11:14:14 epoch 8, step 30, loss: 7.724, global_step: 240
2018-06-20 11:16:04,314:INFO: 2018-06-20 11:14:14 epoch 8, step 30, loss: 7.724, global_step: 240
2018-06-20 11:16:04,955:INFO: ===========validation / test===========
2018-06-20 11:16:04,955:INFO: ===========validation / test===========
2018-06-20 11:16:04,955:INFO: ===========validation / test===========
2018-06-20 11:16:21,242:INFO: 2018-06-20 11:16:17 epoch 9, step 1, loss: 9.854, global_step: 241
2018-06-20 11:16:21,242:INFO: 2018-06-20 11:16:17 epoch 9, step 1, loss: 9.854, global_step: 241
2018-06-20 11:16:21,242:INFO: 2018-06-20 11:16:17 epoch 9, step 1, loss: 9.854, global_step: 241
2018-06-20 11:18:05,933:INFO: 2018-06-20 11:16:17 epoch 9, step 30, loss: 5.046, global_step: 270
2018-06-20 11:18:05,933:INFO: 2018-06-20 11:16:17 epoch 9, step 30, loss: 5.046, global_step: 270
2018-06-20 11:18:05,933:INFO: 2018-06-20 11:16:17 epoch 9, step 30, loss: 5.046, global_step: 270
2018-06-20 11:18:06,497:INFO: ===========validation / test===========
2018-06-20 11:18:06,497:INFO: ===========validation / test===========
2018-06-20 11:18:06,497:INFO: ===========validation / test===========
2018-06-20 11:18:20,934:INFO: 2018-06-20 11:18:19 epoch 10, step 1, loss: 11.17, global_step: 271
2018-06-20 11:18:20,934:INFO: 2018-06-20 11:18:19 epoch 10, step 1, loss: 11.17, global_step: 271
2018-06-20 11:18:20,934:INFO: 2018-06-20 11:18:19 epoch 10, step 1, loss: 11.17, global_step: 271
2018-06-20 11:20:12,556:INFO: 2018-06-20 11:18:19 epoch 10, step 30, loss: 5.749, global_step: 300
2018-06-20 11:20:12,556:INFO: 2018-06-20 11:18:19 epoch 10, step 30, loss: 5.749, global_step: 300
2018-06-20 11:20:12,556:INFO: 2018-06-20 11:18:19 epoch 10, step 30, loss: 5.749, global_step: 300
2018-06-20 11:20:13,201:INFO: ===========validation / test===========
2018-06-20 11:20:13,201:INFO: ===========validation / test===========
2018-06-20 11:20:13,201:INFO: ===========validation / test===========
2018-06-20 11:20:29,264:INFO: 2018-06-20 11:20:26 epoch 11, step 1, loss: 9.538, global_step: 301
2018-06-20 11:20:29,264:INFO: 2018-06-20 11:20:26 epoch 11, step 1, loss: 9.538, global_step: 301
2018-06-20 11:20:29,264:INFO: 2018-06-20 11:20:26 epoch 11, step 1, loss: 9.538, global_step: 301
2018-06-20 11:22:23,609:INFO: 2018-06-20 11:20:26 epoch 11, step 30, loss: 4.057, global_step: 330
2018-06-20 11:22:23,609:INFO: 2018-06-20 11:20:26 epoch 11, step 30, loss: 4.057, global_step: 330
2018-06-20 11:22:23,609:INFO: 2018-06-20 11:20:26 epoch 11, step 30, loss: 4.057, global_step: 330
2018-06-20 11:22:24,298:INFO: ===========validation / test===========
2018-06-20 11:22:24,298:INFO: ===========validation / test===========
2018-06-20 11:22:24,298:INFO: ===========validation / test===========
2018-06-20 11:22:44,071:INFO: 2018-06-20 11:22:38 epoch 12, step 1, loss: 7.43, global_step: 331
2018-06-20 11:22:44,071:INFO: 2018-06-20 11:22:38 epoch 12, step 1, loss: 7.43, global_step: 331
2018-06-20 11:22:44,071:INFO: 2018-06-20 11:22:38 epoch 12, step 1, loss: 7.43, global_step: 331
2018-06-20 11:24:28,956:INFO: 2018-06-20 11:22:38 epoch 12, step 30, loss: 3.672, global_step: 360
2018-06-20 11:24:28,956:INFO: 2018-06-20 11:22:38 epoch 12, step 30, loss: 3.672, global_step: 360
2018-06-20 11:24:28,956:INFO: 2018-06-20 11:22:38 epoch 12, step 30, loss: 3.672, global_step: 360
2018-06-20 11:24:29,736:INFO: ===========validation / test===========
2018-06-20 11:24:29,736:INFO: ===========validation / test===========
2018-06-20 11:24:29,736:INFO: ===========validation / test===========
2018-06-20 11:24:47,606:INFO: 2018-06-20 11:24:42 epoch 13, step 1, loss: 8.37, global_step: 361
2018-06-20 11:24:47,606:INFO: 2018-06-20 11:24:42 epoch 13, step 1, loss: 8.37, global_step: 361
2018-06-20 11:24:47,606:INFO: 2018-06-20 11:24:42 epoch 13, step 1, loss: 8.37, global_step: 361
2018-06-20 11:26:34,757:INFO: 2018-06-20 11:24:42 epoch 13, step 30, loss: 7.59, global_step: 390
2018-06-20 11:26:34,757:INFO: 2018-06-20 11:24:42 epoch 13, step 30, loss: 7.59, global_step: 390
2018-06-20 11:26:34,757:INFO: 2018-06-20 11:24:42 epoch 13, step 30, loss: 7.59, global_step: 390
2018-06-20 11:26:35,680:INFO: ===========validation / test===========
2018-06-20 11:26:35,680:INFO: ===========validation / test===========
2018-06-20 11:26:35,680:INFO: ===========validation / test===========
2018-06-20 11:26:53,782:INFO: 2018-06-20 11:26:49 epoch 14, step 1, loss: 7.703, global_step: 391
2018-06-20 11:26:53,782:INFO: 2018-06-20 11:26:49 epoch 14, step 1, loss: 7.703, global_step: 391
2018-06-20 11:26:53,782:INFO: 2018-06-20 11:26:49 epoch 14, step 1, loss: 7.703, global_step: 391
2018-06-20 11:28:50,275:INFO: 2018-06-20 11:26:49 epoch 14, step 30, loss: 1.521, global_step: 420
2018-06-20 11:28:50,275:INFO: 2018-06-20 11:26:49 epoch 14, step 30, loss: 1.521, global_step: 420
2018-06-20 11:28:50,275:INFO: 2018-06-20 11:26:49 epoch 14, step 30, loss: 1.521, global_step: 420
2018-06-20 11:28:50,839:INFO: ===========validation / test===========
2018-06-20 11:28:50,839:INFO: ===========validation / test===========
2018-06-20 11:28:50,839:INFO: ===========validation / test===========
2018-06-20 11:29:10,306:INFO: 2018-06-20 11:29:04 epoch 15, step 1, loss: 6.109, global_step: 421
2018-06-20 11:29:10,306:INFO: 2018-06-20 11:29:04 epoch 15, step 1, loss: 6.109, global_step: 421
2018-06-20 11:29:10,306:INFO: 2018-06-20 11:29:04 epoch 15, step 1, loss: 6.109, global_step: 421
2018-06-20 11:30:49,458:INFO: 2018-06-20 11:29:04 epoch 15, step 30, loss: 3.748, global_step: 450
2018-06-20 11:30:49,458:INFO: 2018-06-20 11:29:04 epoch 15, step 30, loss: 3.748, global_step: 450
2018-06-20 11:30:49,458:INFO: 2018-06-20 11:29:04 epoch 15, step 30, loss: 3.748, global_step: 450
2018-06-20 11:30:50,342:INFO: ===========validation / test===========
2018-06-20 11:30:50,342:INFO: ===========validation / test===========
2018-06-20 11:30:50,342:INFO: ===========validation / test===========
2018-06-20 11:31:06,065:INFO: 2018-06-20 11:31:03 epoch 16, step 1, loss: 4.659, global_step: 451
2018-06-20 11:31:06,065:INFO: 2018-06-20 11:31:03 epoch 16, step 1, loss: 4.659, global_step: 451
2018-06-20 11:31:06,065:INFO: 2018-06-20 11:31:03 epoch 16, step 1, loss: 4.659, global_step: 451
2018-06-20 11:32:55,053:INFO: 2018-06-20 11:31:03 epoch 16, step 30, loss: 5.699, global_step: 480
2018-06-20 11:32:55,053:INFO: 2018-06-20 11:31:03 epoch 16, step 30, loss: 5.699, global_step: 480
2018-06-20 11:32:55,053:INFO: 2018-06-20 11:31:03 epoch 16, step 30, loss: 5.699, global_step: 480
2018-06-20 11:32:55,603:INFO: ===========validation / test===========
2018-06-20 11:32:55,603:INFO: ===========validation / test===========
2018-06-20 11:32:55,603:INFO: ===========validation / test===========
2018-06-20 11:33:12,443:INFO: 2018-06-20 11:33:08 epoch 17, step 1, loss: 5.268, global_step: 481
2018-06-20 11:33:12,443:INFO: 2018-06-20 11:33:08 epoch 17, step 1, loss: 5.268, global_step: 481
2018-06-20 11:33:12,443:INFO: 2018-06-20 11:33:08 epoch 17, step 1, loss: 5.268, global_step: 481
2018-06-20 11:34:56,772:INFO: 2018-06-20 11:33:08 epoch 17, step 30, loss: 10.84, global_step: 510
2018-06-20 11:34:56,772:INFO: 2018-06-20 11:33:08 epoch 17, step 30, loss: 10.84, global_step: 510
2018-06-20 11:34:56,772:INFO: 2018-06-20 11:33:08 epoch 17, step 30, loss: 10.84, global_step: 510
2018-06-20 11:34:57,406:INFO: ===========validation / test===========
2018-06-20 11:34:57,406:INFO: ===========validation / test===========
2018-06-20 11:34:57,406:INFO: ===========validation / test===========
2018-06-20 11:35:16,564:INFO: 2018-06-20 11:35:10 epoch 18, step 1, loss: 7.437, global_step: 511
2018-06-20 11:35:16,564:INFO: 2018-06-20 11:35:10 epoch 18, step 1, loss: 7.437, global_step: 511
2018-06-20 11:35:16,564:INFO: 2018-06-20 11:35:10 epoch 18, step 1, loss: 7.437, global_step: 511
2018-06-20 11:37:02,349:INFO: 2018-06-20 11:35:10 epoch 18, step 30, loss: 7.158, global_step: 540
2018-06-20 11:37:02,349:INFO: 2018-06-20 11:35:10 epoch 18, step 30, loss: 7.158, global_step: 540
2018-06-20 11:37:02,349:INFO: 2018-06-20 11:35:10 epoch 18, step 30, loss: 7.158, global_step: 540
2018-06-20 11:37:02,867:INFO: ===========validation / test===========
2018-06-20 11:37:02,867:INFO: ===========validation / test===========
2018-06-20 11:37:02,867:INFO: ===========validation / test===========
2018-06-20 11:37:19,666:INFO: 2018-06-20 11:37:16 epoch 19, step 1, loss: 5.713, global_step: 541
2018-06-20 11:37:19,666:INFO: 2018-06-20 11:37:16 epoch 19, step 1, loss: 5.713, global_step: 541
2018-06-20 11:37:19,666:INFO: 2018-06-20 11:37:16 epoch 19, step 1, loss: 5.713, global_step: 541
2018-06-20 11:39:13,803:INFO: 2018-06-20 11:37:16 epoch 19, step 30, loss: 3.75, global_step: 570
2018-06-20 11:39:13,803:INFO: 2018-06-20 11:37:16 epoch 19, step 30, loss: 3.75, global_step: 570
2018-06-20 11:39:13,803:INFO: 2018-06-20 11:37:16 epoch 19, step 30, loss: 3.75, global_step: 570
2018-06-20 11:39:14,418:INFO: ===========validation / test===========
2018-06-20 11:39:14,418:INFO: ===========validation / test===========
2018-06-20 11:39:14,418:INFO: ===========validation / test===========
2018-06-20 11:39:32,268:INFO: 2018-06-20 11:39:28 epoch 20, step 1, loss: 2.383, global_step: 571
2018-06-20 11:39:32,268:INFO: 2018-06-20 11:39:28 epoch 20, step 1, loss: 2.383, global_step: 571
2018-06-20 11:39:32,268:INFO: 2018-06-20 11:39:28 epoch 20, step 1, loss: 2.383, global_step: 571
2018-06-20 11:41:18,893:INFO: 2018-06-20 11:39:28 epoch 20, step 30, loss: 9.322, global_step: 600
2018-06-20 11:41:18,893:INFO: 2018-06-20 11:39:28 epoch 20, step 30, loss: 9.322, global_step: 600
2018-06-20 11:41:18,893:INFO: 2018-06-20 11:39:28 epoch 20, step 30, loss: 9.322, global_step: 600
2018-06-20 11:41:19,464:INFO: ===========validation / test===========
2018-06-20 11:41:19,464:INFO: ===========validation / test===========
2018-06-20 11:41:19,464:INFO: ===========validation / test===========
2018-06-20 11:41:39,209:INFO: 2018-06-20 11:41:33 epoch 21, step 1, loss: 4.853, global_step: 601
2018-06-20 11:41:39,209:INFO: 2018-06-20 11:41:33 epoch 21, step 1, loss: 4.853, global_step: 601
2018-06-20 11:41:39,209:INFO: 2018-06-20 11:41:33 epoch 21, step 1, loss: 4.853, global_step: 601
2018-06-20 11:43:29,862:INFO: 2018-06-20 11:41:33 epoch 21, step 30, loss: 1.345, global_step: 630
2018-06-20 11:43:29,862:INFO: 2018-06-20 11:41:33 epoch 21, step 30, loss: 1.345, global_step: 630
2018-06-20 11:43:29,862:INFO: 2018-06-20 11:41:33 epoch 21, step 30, loss: 1.345, global_step: 630
2018-06-20 11:43:30,524:INFO: ===========validation / test===========
2018-06-20 11:43:30,524:INFO: ===========validation / test===========
2018-06-20 11:43:30,524:INFO: ===========validation / test===========
2018-06-20 11:43:48,949:INFO: 2018-06-20 11:43:44 epoch 22, step 1, loss: 4.871, global_step: 631
2018-06-20 11:43:48,949:INFO: 2018-06-20 11:43:44 epoch 22, step 1, loss: 4.871, global_step: 631
2018-06-20 11:43:48,949:INFO: 2018-06-20 11:43:44 epoch 22, step 1, loss: 4.871, global_step: 631
2018-06-20 11:45:41,291:INFO: 2018-06-20 11:43:44 epoch 22, step 30, loss: 1.35, global_step: 660
2018-06-20 11:45:41,291:INFO: 2018-06-20 11:43:44 epoch 22, step 30, loss: 1.35, global_step: 660
2018-06-20 11:45:41,291:INFO: 2018-06-20 11:43:44 epoch 22, step 30, loss: 1.35, global_step: 660
2018-06-20 11:45:41,899:INFO: ===========validation / test===========
2018-06-20 11:45:41,899:INFO: ===========validation / test===========
2018-06-20 11:45:41,899:INFO: ===========validation / test===========
2018-06-20 11:45:58,248:INFO: 2018-06-20 11:45:55 epoch 23, step 1, loss: 3.304, global_step: 661
2018-06-20 11:45:58,248:INFO: 2018-06-20 11:45:55 epoch 23, step 1, loss: 3.304, global_step: 661
2018-06-20 11:45:58,248:INFO: 2018-06-20 11:45:55 epoch 23, step 1, loss: 3.304, global_step: 661
2018-06-20 11:47:50,618:INFO: 2018-06-20 11:45:55 epoch 23, step 30, loss: 3.188, global_step: 690
2018-06-20 11:47:50,618:INFO: 2018-06-20 11:45:55 epoch 23, step 30, loss: 3.188, global_step: 690
2018-06-20 11:47:50,618:INFO: 2018-06-20 11:45:55 epoch 23, step 30, loss: 3.188, global_step: 690
2018-06-20 11:47:51,395:INFO: ===========validation / test===========
2018-06-20 11:47:51,395:INFO: ===========validation / test===========
2018-06-20 11:47:51,395:INFO: ===========validation / test===========
2018-06-20 11:48:08,311:INFO: 2018-06-20 11:48:04 epoch 24, step 1, loss: 3.581, global_step: 691
2018-06-20 11:48:08,311:INFO: 2018-06-20 11:48:04 epoch 24, step 1, loss: 3.581, global_step: 691
2018-06-20 11:48:08,311:INFO: 2018-06-20 11:48:04 epoch 24, step 1, loss: 3.581, global_step: 691
2018-06-20 11:49:57,720:INFO: 2018-06-20 11:48:04 epoch 24, step 30, loss: 3.092, global_step: 720
2018-06-20 11:49:57,720:INFO: 2018-06-20 11:48:04 epoch 24, step 30, loss: 3.092, global_step: 720
2018-06-20 11:49:57,720:INFO: 2018-06-20 11:48:04 epoch 24, step 30, loss: 3.092, global_step: 720
2018-06-20 11:49:58,426:INFO: ===========validation / test===========
2018-06-20 11:49:58,426:INFO: ===========validation / test===========
2018-06-20 11:49:58,426:INFO: ===========validation / test===========
2018-06-20 11:50:15,886:INFO: 2018-06-20 11:50:12 epoch 25, step 1, loss: 2.848, global_step: 721
2018-06-20 11:50:15,886:INFO: 2018-06-20 11:50:12 epoch 25, step 1, loss: 2.848, global_step: 721
2018-06-20 11:50:15,886:INFO: 2018-06-20 11:50:12 epoch 25, step 1, loss: 2.848, global_step: 721
2018-06-20 11:51:59,613:INFO: 2018-06-20 11:50:12 epoch 25, step 30, loss: 6.531, global_step: 750
2018-06-20 11:51:59,613:INFO: 2018-06-20 11:50:12 epoch 25, step 30, loss: 6.531, global_step: 750
2018-06-20 11:51:59,613:INFO: 2018-06-20 11:50:12 epoch 25, step 30, loss: 6.531, global_step: 750
2018-06-20 11:52:00,213:INFO: ===========validation / test===========
2018-06-20 11:52:00,213:INFO: ===========validation / test===========
2018-06-20 11:52:00,213:INFO: ===========validation / test===========
2018-06-20 11:52:17,035:INFO: 2018-06-20 11:52:13 epoch 26, step 1, loss: 3.713, global_step: 751
2018-06-20 11:52:17,035:INFO: 2018-06-20 11:52:13 epoch 26, step 1, loss: 3.713, global_step: 751
2018-06-20 11:52:17,035:INFO: 2018-06-20 11:52:13 epoch 26, step 1, loss: 3.713, global_step: 751
2018-06-20 11:54:06,192:INFO: 2018-06-20 11:52:13 epoch 26, step 30, loss: 7.829, global_step: 780
2018-06-20 11:54:06,192:INFO: 2018-06-20 11:52:13 epoch 26, step 30, loss: 7.829, global_step: 780
2018-06-20 11:54:06,192:INFO: 2018-06-20 11:52:13 epoch 26, step 30, loss: 7.829, global_step: 780
2018-06-20 11:54:06,864:INFO: ===========validation / test===========
2018-06-20 11:54:06,864:INFO: ===========validation / test===========
2018-06-20 11:54:06,864:INFO: ===========validation / test===========
2018-06-20 11:54:23,422:INFO: 2018-06-20 11:54:20 epoch 27, step 1, loss: 3.699, global_step: 781
2018-06-20 11:54:23,422:INFO: 2018-06-20 11:54:20 epoch 27, step 1, loss: 3.699, global_step: 781
2018-06-20 11:54:23,422:INFO: 2018-06-20 11:54:20 epoch 27, step 1, loss: 3.699, global_step: 781
2018-06-20 11:56:11,587:INFO: 2018-06-20 11:54:20 epoch 27, step 30, loss: 1.33, global_step: 810
2018-06-20 11:56:11,587:INFO: 2018-06-20 11:54:20 epoch 27, step 30, loss: 1.33, global_step: 810
2018-06-20 11:56:11,587:INFO: 2018-06-20 11:54:20 epoch 27, step 30, loss: 1.33, global_step: 810
2018-06-20 11:56:12,150:INFO: ===========validation / test===========
2018-06-20 11:56:12,150:INFO: ===========validation / test===========
2018-06-20 11:56:12,150:INFO: ===========validation / test===========
2018-06-20 11:56:26,843:INFO: 2018-06-20 11:56:25 epoch 28, step 1, loss: 2.075, global_step: 811
2018-06-20 11:56:26,843:INFO: 2018-06-20 11:56:25 epoch 28, step 1, loss: 2.075, global_step: 811
2018-06-20 11:56:26,843:INFO: 2018-06-20 11:56:25 epoch 28, step 1, loss: 2.075, global_step: 811
2018-06-20 11:58:08,982:INFO: 2018-06-20 11:56:25 epoch 28, step 30, loss: 6.59, global_step: 840
2018-06-20 11:58:08,982:INFO: 2018-06-20 11:56:25 epoch 28, step 30, loss: 6.59, global_step: 840
2018-06-20 11:58:08,982:INFO: 2018-06-20 11:56:25 epoch 28, step 30, loss: 6.59, global_step: 840
2018-06-20 11:58:09,569:INFO: ===========validation / test===========
2018-06-20 11:58:09,569:INFO: ===========validation / test===========
2018-06-20 11:58:09,569:INFO: ===========validation / test===========
2018-06-20 11:58:23,864:INFO: 2018-06-20 11:58:22 epoch 29, step 1, loss: 2.188, global_step: 841
2018-06-20 11:58:23,864:INFO: 2018-06-20 11:58:22 epoch 29, step 1, loss: 2.188, global_step: 841
2018-06-20 11:58:23,864:INFO: 2018-06-20 11:58:22 epoch 29, step 1, loss: 2.188, global_step: 841
2018-06-20 12:00:21,244:INFO: 2018-06-20 11:58:22 epoch 29, step 30, loss: 7.308, global_step: 870
2018-06-20 12:00:21,244:INFO: 2018-06-20 11:58:22 epoch 29, step 30, loss: 7.308, global_step: 870
2018-06-20 12:00:21,244:INFO: 2018-06-20 11:58:22 epoch 29, step 30, loss: 7.308, global_step: 870
2018-06-20 12:00:21,805:INFO: ===========validation / test===========
2018-06-20 12:00:21,805:INFO: ===========validation / test===========
2018-06-20 12:00:21,805:INFO: ===========validation / test===========
2018-06-20 12:00:41,083:INFO: 2018-06-20 12:00:35 epoch 30, step 1, loss: 3.268, global_step: 871
2018-06-20 12:00:41,083:INFO: 2018-06-20 12:00:35 epoch 30, step 1, loss: 3.268, global_step: 871
2018-06-20 12:00:41,083:INFO: 2018-06-20 12:00:35 epoch 30, step 1, loss: 3.268, global_step: 871
2018-06-20 12:02:27,724:INFO: 2018-06-20 12:00:35 epoch 30, step 30, loss: 1.006, global_step: 900
2018-06-20 12:02:27,724:INFO: 2018-06-20 12:00:35 epoch 30, step 30, loss: 1.006, global_step: 900
2018-06-20 12:02:27,724:INFO: 2018-06-20 12:00:35 epoch 30, step 30, loss: 1.006, global_step: 900
2018-06-20 12:02:28,350:INFO: ===========validation / test===========
2018-06-20 12:02:28,350:INFO: ===========validation / test===========
2018-06-20 12:02:28,350:INFO: ===========validation / test===========
2018-06-20 12:02:44,945:INFO: 2018-06-20 12:02:42 epoch 31, step 1, loss: 2.995, global_step: 901
2018-06-20 12:02:44,945:INFO: 2018-06-20 12:02:42 epoch 31, step 1, loss: 2.995, global_step: 901
2018-06-20 12:02:44,945:INFO: 2018-06-20 12:02:42 epoch 31, step 1, loss: 2.995, global_step: 901
2018-06-20 12:04:34,370:INFO: 2018-06-20 12:02:42 epoch 31, step 30, loss: 6.774, global_step: 930
2018-06-20 12:04:34,370:INFO: 2018-06-20 12:02:42 epoch 31, step 30, loss: 6.774, global_step: 930
2018-06-20 12:04:34,370:INFO: 2018-06-20 12:02:42 epoch 31, step 30, loss: 6.774, global_step: 930
2018-06-20 12:04:34,974:INFO: ===========validation / test===========
2018-06-20 12:04:34,974:INFO: ===========validation / test===========
2018-06-20 12:04:34,974:INFO: ===========validation / test===========
2018-06-20 12:04:51,019:INFO: 2018-06-20 12:04:48 epoch 32, step 1, loss: 3.775, global_step: 931
2018-06-20 12:04:51,019:INFO: 2018-06-20 12:04:48 epoch 32, step 1, loss: 3.775, global_step: 931
2018-06-20 12:04:51,019:INFO: 2018-06-20 12:04:48 epoch 32, step 1, loss: 3.775, global_step: 931
2018-06-20 12:06:40,195:INFO: 2018-06-20 12:04:48 epoch 32, step 30, loss: 4.429, global_step: 960
2018-06-20 12:06:40,195:INFO: 2018-06-20 12:04:48 epoch 32, step 30, loss: 4.429, global_step: 960
2018-06-20 12:06:40,195:INFO: 2018-06-20 12:04:48 epoch 32, step 30, loss: 4.429, global_step: 960
2018-06-20 12:06:40,758:INFO: ===========validation / test===========
2018-06-20 12:06:40,758:INFO: ===========validation / test===========
2018-06-20 12:06:40,758:INFO: ===========validation / test===========
2018-06-20 12:06:56,721:INFO: 2018-06-20 12:06:54 epoch 33, step 1, loss: 3.445, global_step: 961
2018-06-20 12:06:56,721:INFO: 2018-06-20 12:06:54 epoch 33, step 1, loss: 3.445, global_step: 961
2018-06-20 12:06:56,721:INFO: 2018-06-20 12:06:54 epoch 33, step 1, loss: 3.445, global_step: 961
2018-06-20 12:08:50,823:INFO: 2018-06-20 12:06:54 epoch 33, step 30, loss: 2.427, global_step: 990
2018-06-20 12:08:50,823:INFO: 2018-06-20 12:06:54 epoch 33, step 30, loss: 2.427, global_step: 990
2018-06-20 12:08:50,823:INFO: 2018-06-20 12:06:54 epoch 33, step 30, loss: 2.427, global_step: 990
2018-06-20 12:08:51,528:INFO: ===========validation / test===========
2018-06-20 12:08:51,528:INFO: ===========validation / test===========
2018-06-20 12:08:51,528:INFO: ===========validation / test===========
2018-06-20 12:09:07,432:INFO: 2018-06-20 12:09:05 epoch 34, step 1, loss: 2.3, global_step: 991
2018-06-20 12:09:07,432:INFO: 2018-06-20 12:09:05 epoch 34, step 1, loss: 2.3, global_step: 991
2018-06-20 12:09:07,432:INFO: 2018-06-20 12:09:05 epoch 34, step 1, loss: 2.3, global_step: 991
2018-06-20 12:10:54,792:INFO: 2018-06-20 12:09:05 epoch 34, step 30, loss: 4.02, global_step: 1020
2018-06-20 12:10:54,792:INFO: 2018-06-20 12:09:05 epoch 34, step 30, loss: 4.02, global_step: 1020
2018-06-20 12:10:54,792:INFO: 2018-06-20 12:09:05 epoch 34, step 30, loss: 4.02, global_step: 1020
2018-06-20 12:10:55,358:INFO: ===========validation / test===========
2018-06-20 12:10:55,358:INFO: ===========validation / test===========
2018-06-20 12:10:55,358:INFO: ===========validation / test===========
2018-06-20 12:11:10,681:INFO: 2018-06-20 12:11:08 epoch 35, step 1, loss: 2.189, global_step: 1021
2018-06-20 12:11:10,681:INFO: 2018-06-20 12:11:08 epoch 35, step 1, loss: 2.189, global_step: 1021
2018-06-20 12:11:10,681:INFO: 2018-06-20 12:11:08 epoch 35, step 1, loss: 2.189, global_step: 1021
2018-06-20 12:13:06,365:INFO: 2018-06-20 12:11:08 epoch 35, step 30, loss: 3.11, global_step: 1050
2018-06-20 12:13:06,365:INFO: 2018-06-20 12:11:08 epoch 35, step 30, loss: 3.11, global_step: 1050
2018-06-20 12:13:06,365:INFO: 2018-06-20 12:11:08 epoch 35, step 30, loss: 3.11, global_step: 1050
2018-06-20 12:13:06,953:INFO: ===========validation / test===========
2018-06-20 12:13:06,953:INFO: ===========validation / test===========
2018-06-20 12:13:06,953:INFO: ===========validation / test===========
2018-06-20 12:13:24,093:INFO: 2018-06-20 12:13:20 epoch 36, step 1, loss: 2.569, global_step: 1051
2018-06-20 12:13:24,093:INFO: 2018-06-20 12:13:20 epoch 36, step 1, loss: 2.569, global_step: 1051
2018-06-20 12:13:24,093:INFO: 2018-06-20 12:13:20 epoch 36, step 1, loss: 2.569, global_step: 1051
2018-06-20 12:15:16,972:INFO: 2018-06-20 12:13:20 epoch 36, step 30, loss: 1.215, global_step: 1080
2018-06-20 12:15:16,972:INFO: 2018-06-20 12:13:20 epoch 36, step 30, loss: 1.215, global_step: 1080
2018-06-20 12:15:16,972:INFO: 2018-06-20 12:13:20 epoch 36, step 30, loss: 1.215, global_step: 1080
2018-06-20 12:15:17,611:INFO: ===========validation / test===========
2018-06-20 12:15:17,611:INFO: ===========validation / test===========
2018-06-20 12:15:17,611:INFO: ===========validation / test===========
2018-06-20 12:15:36,002:INFO: 2018-06-20 12:15:31 epoch 37, step 1, loss: 2.987, global_step: 1081
2018-06-20 12:15:36,002:INFO: 2018-06-20 12:15:31 epoch 37, step 1, loss: 2.987, global_step: 1081
2018-06-20 12:15:36,002:INFO: 2018-06-20 12:15:31 epoch 37, step 1, loss: 2.987, global_step: 1081
2018-06-20 12:17:25,279:INFO: 2018-06-20 12:15:31 epoch 37, step 30, loss: 2.444, global_step: 1110
2018-06-20 12:17:25,279:INFO: 2018-06-20 12:15:31 epoch 37, step 30, loss: 2.444, global_step: 1110
2018-06-20 12:17:25,279:INFO: 2018-06-20 12:15:31 epoch 37, step 30, loss: 2.444, global_step: 1110
2018-06-20 12:17:25,871:INFO: ===========validation / test===========
2018-06-20 12:17:25,871:INFO: ===========validation / test===========
2018-06-20 12:17:25,871:INFO: ===========validation / test===========
2018-06-20 12:17:41,248:INFO: 2018-06-20 12:17:39 epoch 38, step 1, loss: 2.112, global_step: 1111
2018-06-20 12:17:41,248:INFO: 2018-06-20 12:17:39 epoch 38, step 1, loss: 2.112, global_step: 1111
2018-06-20 12:17:41,248:INFO: 2018-06-20 12:17:39 epoch 38, step 1, loss: 2.112, global_step: 1111
2018-06-20 12:19:31,652:INFO: 2018-06-20 12:17:39 epoch 38, step 30, loss: 0.664, global_step: 1140
2018-06-20 12:19:31,652:INFO: 2018-06-20 12:17:39 epoch 38, step 30, loss: 0.664, global_step: 1140
2018-06-20 12:19:31,652:INFO: 2018-06-20 12:17:39 epoch 38, step 30, loss: 0.664, global_step: 1140
2018-06-20 12:19:32,238:INFO: ===========validation / test===========
2018-06-20 12:19:32,238:INFO: ===========validation / test===========
2018-06-20 12:19:32,238:INFO: ===========validation / test===========
2018-06-20 12:19:48,568:INFO: 2018-06-20 12:19:45 epoch 39, step 1, loss: 3.273, global_step: 1141
2018-06-20 12:19:48,568:INFO: 2018-06-20 12:19:45 epoch 39, step 1, loss: 3.273, global_step: 1141
2018-06-20 12:19:48,568:INFO: 2018-06-20 12:19:45 epoch 39, step 1, loss: 3.273, global_step: 1141
2018-06-20 12:21:46,480:INFO: 2018-06-20 12:19:45 epoch 39, step 30, loss: 2.888, global_step: 1170
2018-06-20 12:21:46,480:INFO: 2018-06-20 12:19:45 epoch 39, step 30, loss: 2.888, global_step: 1170
2018-06-20 12:21:46,480:INFO: 2018-06-20 12:19:45 epoch 39, step 30, loss: 2.888, global_step: 1170
2018-06-20 12:21:47,102:INFO: ===========validation / test===========
2018-06-20 12:21:47,102:INFO: ===========validation / test===========
2018-06-20 12:21:47,102:INFO: ===========validation / test===========
2018-06-20 12:22:07,551:INFO: 2018-06-20 12:22:01 epoch 40, step 1, loss: 2.022, global_step: 1171
2018-06-20 12:22:07,551:INFO: 2018-06-20 12:22:01 epoch 40, step 1, loss: 2.022, global_step: 1171
2018-06-20 12:22:07,551:INFO: 2018-06-20 12:22:01 epoch 40, step 1, loss: 2.022, global_step: 1171
2018-06-20 12:23:53,824:INFO: 2018-06-20 12:22:01 epoch 40, step 30, loss: 1.259, global_step: 1200
2018-06-20 12:23:53,824:INFO: 2018-06-20 12:22:01 epoch 40, step 30, loss: 1.259, global_step: 1200
2018-06-20 12:23:53,824:INFO: 2018-06-20 12:22:01 epoch 40, step 30, loss: 1.259, global_step: 1200
2018-06-20 12:23:54,383:INFO: ===========validation / test===========
2018-06-20 12:23:54,383:INFO: ===========validation / test===========
2018-06-20 12:23:54,383:INFO: ===========validation / test===========
2018-06-20 12:28:27,233:INFO: Restoring parameters from .\data_path_save\1529463214\checkpoints/model-1200
2018-06-20 12:38:50,743:INFO: Restoring parameters from .\data_path_save\1529463214\checkpoints/model-1200
2018-06-21 10:11:17,999:INFO: Restoring parameters from .\data_path_save\1529463214\checkpoints/model-1200
2018-06-27 11:43:33,271:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 11:46:44,915:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 11:48:52,346:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 11:49:02,627:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 48, in list
    demo_sent = list(s.strip())
  File "D:\github\web\mysite\webshow\views.py", line 48, in list
    demo_sent = list(s.strip())
  File "D:\github\web\mysite\webshow\views.py", line 48, in list
    demo_sent = list(s.strip())
  [Previous line repeated 934 more times]
RecursionError: maximum recursion depth exceeded
2018-06-27 11:49:06,302:WARNING: Not Found: /favicon.ico
2018-06-27 11:54:10,962:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 11:54:25,294:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 53, in list
    return render(request, 'list.html', s)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\backends\django.py", line 59, in render
    context = make_context(context, request, autoescape=self.backend.engine.autoescape)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\context.py", line 274, in make_context
    raise TypeError('context must be a dict rather than %s.' % context.__class__.__name__)
TypeError: context must be a dict rather than str.
2018-06-27 11:54:25,805:WARNING: Not Found: /favicon.ico
2018-06-27 11:57:19,884:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 11:57:26,847:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 11:57:26,851:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 11:57:26,871:WARNING: Not Found: /favicon.ico
2018-06-27 11:57:50,864:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 11:57:50,867:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 11:57:50,872:WARNING: Not Found: /favicon.ico
2018-06-27 11:58:04,081:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 11:58:04,102:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 11:58:04,106:WARNING: Not Found: /favicon.ico
2018-06-27 11:58:28,594:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 11:58:28,597:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 11:58:28,599:WARNING: Not Found: /favicon.ico
2018-06-27 11:59:16,809:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 475, in parse
    compile_func = self.tags[command]
KeyError: "tag['s']"

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 55, in list
    return render(request, 'list.html', tag)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 198, in compile_nodelist
    return parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 477, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 533, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 5: 'tag['s']', expected 'endblock'. Did you forget to register or load this tag?
2018-06-27 11:59:17,394:WARNING: Not Found: /favicon.ico
2018-06-27 12:01:15,116:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:01:15,120:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:01:15,166:WARNING: Not Found: /favicon.ico
2018-06-27 12:01:42,560:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:01:42,568:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:01:42,575:WARNING: Not Found: /favicon.ico
2018-06-27 12:01:45,632:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:01:45,635:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:01:45,638:WARNING: Not Found: /favicon.ico
2018-06-27 12:02:22,225:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:02:22,236:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:02:22,237:WARNING: Not Found: /favicon.ico
2018-06-27 12:03:12,399:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:03:12,406:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:03:12,414:WARNING: Not Found: /favicon.ico
2018-06-27 12:03:13,990:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:03:13,994:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:03:14,008:WARNING: Not Found: /favicon.ico
2018-06-27 12:05:20,062:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:05:20,069:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:05:20,076:WARNING: Not Found: /favicon.ico
2018-06-27 12:06:09,709:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 12:06:22,046:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 50, in list
    demo_sent = list(s.strip())
  File "D:\github\web\mysite\webshow\views.py", line 50, in list
    demo_sent = list(s.strip())
  File "D:\github\web\mysite\webshow\views.py", line 50, in list
    demo_sent = list(s.strip())
  [Previous line repeated 933 more times]
  File "D:\github\web\mysite\webshow\views.py", line 48, in list
    tag = dict()
RecursionError: maximum recursion depth exceeded while calling a Python object
2018-06-27 12:06:25,738:WARNING: Not Found: /favicon.ico
2018-06-27 12:10:45,163:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 12:11:02,819:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 12:11:02,822:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 12:11:02,842:WARNING: Not Found: /favicon.ico
2018-06-27 12:19:02,361:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 14:13:33,221:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 14:13:59,728:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1322, in _do_call
    return fn(*args)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1307, in _run_fn
    options, feed_dict, fetch_list, target_list, run_metadata)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1409, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.InternalError: Blas GEMM launch failed : a.shape=(1, 600), b.shape=(600, 1200), m=1, n=1200, k=600
	 [[Node: bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul = MatMul[T=DT_FLOAT, transpose_a=false, transpose_b=false, _device="/job:localhost/replica:0/task:0/device:GPU:0"](bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/concat, bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul/Enter)]]
	 [[Node: proj/Reshape_1/_103 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_399_proj/Reshape_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 50, in list1
    tag = model.demo_one(sess, demo_data)
  File "D:\github\web\mysite\webshow\ner\model.py", line 192, in demo_one
    label_list_, _ = self.predict_one_batch(sess, seqs)
  File "D:\github\web\mysite\webshow\ner\model.py", line 285, in predict_one_batch
    feed_dict=feed_dict)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 900, in run
    run_metadata_ptr)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1135, in _run
    feed_dict_tensor, options, run_metadata)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1316, in _do_run
    run_metadata)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1335, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.InternalError: Blas GEMM launch failed : a.shape=(1, 600), b.shape=(600, 1200), m=1, n=1200, k=600
	 [[Node: bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul = MatMul[T=DT_FLOAT, transpose_a=false, transpose_b=false, _device="/job:localhost/replica:0/task:0/device:GPU:0"](bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/concat, bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul/Enter)]]
	 [[Node: proj/Reshape_1/_103 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_399_proj/Reshape_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

Caused by op 'bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul', defined at:
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\utils\autoreload.py", line 225, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\management\commands\runserver.py", line 120, in inner_run
    self.check(display_num_errors=True)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\management\base.py", line 364, in check
    include_deployment_checks=include_deployment_checks,
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\management\base.py", line 351, in _run_checks
    return checks.run_checks(**kwargs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\checks\registry.py", line 73, in run_checks
    new_errors = check(app_configs=app_configs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\checks\urls.py", line 40, in check_url_namespaces_unique
    all_namespaces = _load_all_namespaces(resolver)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\checks\urls.py", line 57, in _load_all_namespaces
    url_patterns = getattr(resolver, 'url_patterns', [])
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\utils\functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\urls\resolvers.py", line 536, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\utils\functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\urls\resolvers.py", line 529, in urlconf_module
    return import_module(self.urlconf_name)
  File "C:\Users\Ian\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\github\web\mysite\mysite\urls.py", line 32, in <module>
    path('webshow/', include('webshow.urls')),
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\urls\conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "C:\Users\Ian\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\github\web\mysite\webshow\urls.py", line 3, in <module>
    from . import views
  File "<frozen importlib._bootstrap>", line 1023, in _handle_fromlist
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\github\web\mysite\webshow\views.py", line 40, in <module>
    model.build_graph()
  File "D:\github\web\mysite\webshow\ner\model.py", line 49, in build_graph
    self.biLSTM_layer_op()
  File "D:\github\web\mysite\webshow\ner\model.py", line 83, in biLSTM_layer_op
    dtype=tf.float32)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 438, in bidirectional_dynamic_rnn
    time_major=time_major, scope=bw_scope)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 627, in dynamic_rnn
    dtype=dtype)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 824, in _dynamic_rnn_loop
    swap_memory=swap_memory)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\control_flow_ops.py", line 3224, in while_loop
    result = loop_context.BuildLoop(cond, body, loop_vars, shape_invariants)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\control_flow_ops.py", line 2956, in BuildLoop
    pred, body, original_loop_vars, loop_vars, shape_invariants)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\control_flow_ops.py", line 2893, in _BuildLoop
    body_result = body(*packed_vars_for_body)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\control_flow_ops.py", line 3194, in <lambda>
    body = lambda i, lv: (i + 1, orig_body(*lv))
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 793, in _time_step
    skip_conditionals=True)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 248, in _rnn_step
    new_output, new_state = call_cell()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 781, in <lambda>
    call_cell = lambda: cell(input_t, state)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn_cell_impl.py", line 339, in __call__
    *args, **kwargs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\layers\base.py", line 717, in __call__
    outputs = self.call(inputs, *args, **kwargs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn_cell_impl.py", line 833, in call
    array_ops.concat([inputs, m_prev], 1), self._kernel)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\math_ops.py", line 2122, in matmul
    a, b, transpose_a=transpose_a, transpose_b=transpose_b, name=name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_math_ops.py", line 4567, in mat_mul
    name=name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3392, in create_op
    op_def=op_def)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 1718, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

InternalError (see above for traceback): Blas GEMM launch failed : a.shape=(1, 600), b.shape=(600, 1200), m=1, n=1200, k=600
	 [[Node: bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul = MatMul[T=DT_FLOAT, transpose_a=false, transpose_b=false, _device="/job:localhost/replica:0/task:0/device:GPU:0"](bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/concat, bi-lstm/bidirectional_rnn/bw/bw/while/lstm_cell/MatMul/Enter)]]
	 [[Node: proj/Reshape_1/_103 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_399_proj/Reshape_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

2018-06-27 14:14:00,522:WARNING: Not Found: /favicon.ico
2018-06-27 14:15:48,107:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 14:16:01,512:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 14:16:01,517:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 14:16:01,545:WARNING: Not Found: /favicon.ico
2018-06-27 14:17:55,096:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 14:17:55,101:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 14:17:55,104:WARNING: Not Found: /favicon.ico
2018-06-27 14:34:20,926:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 14:38:54,432:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 14:39:38,406:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 53, in list1
    return render(request, 'list.html', l)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 198, in compile_nodelist
    return parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\defaulttags.py", line 815, in do_for
    nodelist_loop = parser.parse(('empty', 'endfor',))
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 454, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 452, in parse
    filter_expression = self.compile_filter(token.contents)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 568, in compile_filter
    return FilterExpression(token, self)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 668, in __init__
    "from '%s'" % (token[upto:], token))
django.template.exceptions.TemplateSyntaxError: Could not parse the remainder: '{ foo.1' from '{ foo.1'
2018-06-27 14:39:39,056:WARNING: Not Found: /favicon.ico
2018-06-27 14:43:27,311:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 53, in list1
    return render(request, 'list.html', l)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\backends\django.py", line 59, in render
    context = make_context(context, request, autoescape=self.backend.engine.autoescape)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\context.py", line 274, in make_context
    raise TypeError('context must be a dict rather than %s.' % context.__class__.__name__)
TypeError: context must be a dict rather than list.
2018-06-27 14:43:27,458:WARNING: Not Found: /favicon.ico
2018-06-27 14:49:12,537:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 14:49:15,502:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 475, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 53, in list1
    return render(request, 'list.html', d)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 198, in compile_nodelist
    return parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 477, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 533, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 47: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
2018-06-27 14:49:16,015:WARNING: Not Found: /favicon.ico
2018-06-27 14:49:20,659:ERROR: Internal Server Error: /webshow/list/
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 475, in parse
    compile_func = self.tags[command]
KeyError: 'endfor'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 53, in list1
    return render(request, 'list.html', d)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\shortcuts.py", line 36, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader.py", line 15, in get_template
    return engine.get_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\backends\django.py", line 34, in get_template
    return Template(self.engine.get_template(template_name), self)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 144, in get_template
    template, origin = self.find_template(template_name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\engine.py", line 126, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loaders\base.py", line 30, in get_template
    contents, origin, origin.template_name, self.engine,
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 160, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 198, in compile_nodelist
    return parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 290, in do_extends
    nodelist = parser.parse()
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 483, in parse
    raise self.error(token, e)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 481, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\loader_tags.py", line 233, in do_block
    nodelist = parser.parse(('endblock',))
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 477, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\template\base.py", line 533, in invalid_block_tag
    get_text_list(["'%s'" % p for p in parse_until], 'or'),
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 47: 'endfor', expected 'endblock'. Did you forget to register or load this tag?
2018-06-27 14:49:20,884:WARNING: Not Found: /favicon.ico
2018-06-27 14:50:01,060:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 14:50:01,064:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 14:50:01,091:WARNING: Not Found: /favicon.ico
2018-06-27 14:53:01,726:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 14:53:01,732:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 14:53:01,737:WARNING: Not Found: /favicon.ico
2018-06-27 14:53:23,449:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 14:53:23,454:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 14:53:23,458:WARNING: Not Found: /favicon.ico
2018-06-27 15:03:25,565:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 15:05:36,810:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 15:06:10,426:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 15:07:03,238:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:07:03,242:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:07:03,258:WARNING: Not Found: /favicon.ico
2018-06-27 15:09:15,224:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:09:15,243:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:09:15,254:WARNING: Not Found: /favicon.ico
2018-06-27 15:10:17,022:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:10:17,059:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:10:17,085:WARNING: Not Found: /favicon.ico
2018-06-27 15:12:57,538:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:12:57,542:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:12:57,576:WARNING: Not Found: /favicon.ico
2018-06-27 15:13:45,112:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:13:45,122:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:13:45,148:WARNING: Not Found: /favicon.ico
2018-06-27 15:13:47,903:WARNING: Forbidden (CSRF token missing or incorrect.): /webshow/list/ner
2018-06-27 15:14:28,092:WARNING: Forbidden (CSRF token missing or incorrect.): /webshow/list/ner
2018-06-27 15:15:27,331:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:15:27,365:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:15:27,368:WARNING: Not Found: /favicon.ico
2018-06-27 15:15:48,884:WARNING: Forbidden (CSRF token missing or incorrect.): /webshow/list/ner
2018-06-27 15:17:36,719:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 15:17:47,031:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:17:47,055:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:17:47,069:WARNING: Not Found: /favicon.ico
2018-06-27 15:17:50,479:WARNING: Forbidden (CSRF token missing or incorrect.): /webshow/list/ner
2018-06-27 15:24:02,275:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:24:02,299:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:24:02,301:WARNING: Not Found: /favicon.ico
2018-06-27 15:24:31,851:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:24:31,906:WARNING: Not Found: /favicon.ico
2018-06-27 15:24:35,220:WARNING: Forbidden (CSRF token missing or incorrect.): /webshow/list/ner
2018-06-27 15:26:48,630:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:26:48,634:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:26:48,645:WARNING: Not Found: /favicon.ico
2018-06-27 15:26:50,952:ERROR: Internal Server Error: /webshow/list/ner
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1322, in _do_call
    return fn(*args)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1307, in _run_fn
    options, feed_dict, fetch_list, target_list, run_metadata)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1409, in _call_tf_sessionrun
    run_metadata)
tensorflow.python.framework.errors_impl.UnimplementedError: TensorArray has size zero, but element shape [?,300] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[dtype=DT_FLOAT, element_shape=[?,300], _device="/job:localhost/replica:0/task:0/device:GPU:0"](bi-lstm/bidirectional_rnn/bw/bw/TensorArray, bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/range, bi-lstm/bidirectional_rnn/bw/bw/while/Exit_2)]]
	 [[Node: proj/Reshape_1/_103 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_399_proj/Reshape_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 60, in ner
    tag = model.demo_one(sess, demo_data)
  File "D:\github\web\mysite\webshow\ner\model.py", line 192, in demo_one
    label_list_, _ = self.predict_one_batch(sess, seqs)
  File "D:\github\web\mysite\webshow\ner\model.py", line 285, in predict_one_batch
    feed_dict=feed_dict)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 900, in run
    run_metadata_ptr)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1135, in _run
    feed_dict_tensor, options, run_metadata)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1316, in _do_run
    run_metadata)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\client\session.py", line 1335, in _do_call
    raise type(e)(node_def, op, message)
tensorflow.python.framework.errors_impl.UnimplementedError: TensorArray has size zero, but element shape [?,300] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[dtype=DT_FLOAT, element_shape=[?,300], _device="/job:localhost/replica:0/task:0/device:GPU:0"](bi-lstm/bidirectional_rnn/bw/bw/TensorArray, bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/range, bi-lstm/bidirectional_rnn/bw/bw/while/Exit_2)]]
	 [[Node: proj/Reshape_1/_103 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_399_proj/Reshape_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

Caused by op 'bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/TensorArrayGatherV3', defined at:
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\utils\autoreload.py", line 225, in wrapper
    fn(*args, **kwargs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\management\commands\runserver.py", line 120, in inner_run
    self.check(display_num_errors=True)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\management\base.py", line 364, in check
    include_deployment_checks=include_deployment_checks,
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\management\base.py", line 351, in _run_checks
    return checks.run_checks(**kwargs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\checks\registry.py", line 73, in run_checks
    new_errors = check(app_configs=app_configs)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\checks\urls.py", line 40, in check_url_namespaces_unique
    all_namespaces = _load_all_namespaces(resolver)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\checks\urls.py", line 57, in _load_all_namespaces
    url_patterns = getattr(resolver, 'url_patterns', [])
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\utils\functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\urls\resolvers.py", line 536, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\utils\functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\urls\resolvers.py", line 529, in urlconf_module
    return import_module(self.urlconf_name)
  File "C:\Users\Ian\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\github\web\mysite\mysite\urls.py", line 32, in <module>
    path('webshow/', include('webshow.urls')),
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\urls\conf.py", line 34, in include
    urlconf_module = import_module(urlconf_module)
  File "C:\Users\Ian\Anaconda3\lib\importlib\__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 994, in _gcd_import
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\github\web\mysite\webshow\urls.py", line 3, in <module>
    from . import views
  File "<frozen importlib._bootstrap>", line 1023, in _handle_fromlist
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "<frozen importlib._bootstrap>", line 971, in _find_and_load
  File "<frozen importlib._bootstrap>", line 955, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 665, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "D:\github\web\mysite\webshow\views.py", line 42, in <module>
    model.build_graph()
  File "D:\github\web\mysite\webshow\ner\model.py", line 49, in build_graph
    self.biLSTM_layer_op()
  File "D:\github\web\mysite\webshow\ner\model.py", line 83, in biLSTM_layer_op
    dtype=tf.float32)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 438, in bidirectional_dynamic_rnn
    time_major=time_major, scope=bw_scope)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 627, in dynamic_rnn
    dtype=dtype)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 828, in _dynamic_rnn_loop
    final_outputs = tuple(ta.stack() for ta in output_final_ta)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\rnn.py", line 828, in <genexpr>
    final_outputs = tuple(ta.stack() for ta in output_final_ta)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\tensor_array_ops.py", line 893, in stack
    return self._implementation.stack(name=name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\tensor_array_ops.py", line 291, in stack
    return self.gather(math_ops.range(0, self.size()), name=name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\tensor_array_ops.py", line 305, in gather
    element_shape=element_shape)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\ops\gen_data_flow_ops.py", line 6479, in tensor_array_gather_v3
    flow_in=flow_in, dtype=dtype, element_shape=element_shape, name=name)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\framework\op_def_library.py", line 787, in _apply_op_helper
    op_def=op_def)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 3392, in create_op
    op_def=op_def)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\tensorflow\python\framework\ops.py", line 1718, in __init__
    self._traceback = self._graph._extract_stack()  # pylint: disable=protected-access

UnimplementedError (see above for traceback): TensorArray has size zero, but element shape [?,300] is not fully defined. Currently only static shapes are supported when packing zero-size TensorArrays.
	 [[Node: bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/TensorArrayGatherV3 = TensorArrayGatherV3[dtype=DT_FLOAT, element_shape=[?,300], _device="/job:localhost/replica:0/task:0/device:GPU:0"](bi-lstm/bidirectional_rnn/bw/bw/TensorArray, bi-lstm/bidirectional_rnn/bw/bw/TensorArrayStack/range, bi-lstm/bidirectional_rnn/bw/bw/while/Exit_2)]]
	 [[Node: proj/Reshape_1/_103 = _Recv[client_terminated=false, recv_device="/job:localhost/replica:0/task:0/device:CPU:0", send_device="/job:localhost/replica:0/task:0/device:GPU:0", send_device_incarnation=1, tensor_name="edge_399_proj/Reshape_1", tensor_type=DT_FLOAT, _device="/job:localhost/replica:0/task:0/device:CPU:0"]()]]

2018-06-27 15:30:52,309:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 15:30:59,058:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:30:59,064:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:30:59,101:WARNING: Not Found: /favicon.ico
2018-06-27 15:57:00,241:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:57:00,247:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:57:00,267:WARNING: Not Found: /favicon.ico
2018-06-27 15:57:45,339:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:57:45,345:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:57:45,360:WARNING: Not Found: /favicon.ico
2018-06-27 15:58:10,905:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 15:58:10,909:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 15:58:10,932:WARNING: Not Found: /favicon.ico
2018-06-27 16:01:07,087:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:01:07,092:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:01:07,121:WARNING: Not Found: /favicon.ico
2018-06-27 16:01:45,462:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:01:45,485:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:01:45,490:WARNING: Not Found: /favicon.ico
2018-06-27 16:02:16,809:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:02:16,814:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:02:16,839:WARNING: Not Found: /favicon.ico
2018-06-27 16:03:12,620:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:03:12,624:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:03:12,630:WARNING: Not Found: /favicon.ico
2018-06-27 16:07:30,083:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:07:30,094:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:07:30,098:WARNING: Not Found: /favicon.ico
2018-06-27 16:08:13,662:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:08:13,670:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:08:13,696:WARNING: Not Found: /favicon.ico
2018-06-27 16:08:35,251:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:08:35,256:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:08:35,289:WARNING: Not Found: /favicon.ico
2018-06-27 16:10:59,988:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:10:59,995:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:11:00,028:WARNING: Not Found: /favicon.ico
2018-06-27 16:13:33,358:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:13:33,363:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:13:33,390:WARNING: Not Found: /favicon.ico
2018-06-27 16:15:00,101:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:15:00,117:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:15:00,138:WARNING: Not Found: /favicon.ico
2018-06-27 16:15:30,474:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:15:30,477:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:15:30,515:WARNING: Not Found: /favicon.ico
2018-06-27 16:16:02,366:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:16:02,377:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:16:02,381:WARNING: Not Found: /favicon.ico
2018-06-27 16:16:25,853:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:16:25,867:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:16:25,871:WARNING: Not Found: /favicon.ico
2018-06-27 16:18:29,464:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:18:29,467:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:18:29,504:WARNING: Not Found: /favicon.ico
2018-06-27 16:18:49,066:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:18:49,075:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:18:49,095:WARNING: Not Found: /favicon.ico
2018-06-27 16:20:22,463:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:20:22,473:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:20:22,498:WARNING: Not Found: /favicon.ico
2018-06-27 16:23:04,644:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:23:04,648:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:23:04,678:WARNING: Not Found: /favicon.ico
2018-06-27 16:26:21,661:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:26:21,689:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:26:21,703:WARNING: Not Found: /favicon.ico
2018-06-27 16:28:34,473:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 16:28:34,475:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 16:28:34,521:WARNING: Not Found: /favicon.ico
2018-06-27 17:07:38,563:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 17:07:38,592:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 17:07:38,597:WARNING: Not Found: /favicon.ico
2018-06-27 17:08:08,485:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 17:08:08,492:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 17:08:08,520:WARNING: Not Found: /favicon.ico
2018-06-27 17:13:05,802:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 17:13:58,961:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 17:15:09,545:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 17:15:09,547:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 17:15:09,563:WARNING: Not Found: /favicon.ico
2018-06-27 17:15:11,854:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 17:15:11,857:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 17:15:11,892:WARNING: Not Found: /favicon.ico
2018-06-27 17:18:35,944:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 17:18:35,948:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 17:18:35,963:WARNING: Not Found: /favicon.ico
2018-06-27 17:23:22,846:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 17:23:22,851:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 17:23:22,873:WARNING: Not Found: /favicon.ico
2018-06-27 18:05:50,142:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-27 18:06:48,247:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:06:48,264:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:06:48,270:WARNING: Not Found: /favicon.ico
2018-06-27 18:33:18,953:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:33:18,958:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:33:18,970:WARNING: Not Found: /favicon.ico
2018-06-27 18:36:20,867:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:36:20,870:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:36:20,885:WARNING: Not Found: /favicon.ico
2018-06-27 18:38:36,826:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:38:36,828:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:38:36,843:WARNING: Not Found: /favicon.ico
2018-06-27 18:38:41,859:WARNING: Not Found: /favicon.ico
2018-06-27 18:43:10,443:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:43:10,445:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:43:10,465:WARNING: Not Found: /favicon.ico
2018-06-27 18:43:48,225:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:43:48,246:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:43:48,260:WARNING: Not Found: /favicon.ico
2018-06-27 18:44:19,749:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:44:19,765:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:44:19,769:WARNING: Not Found: /favicon.ico
2018-06-27 18:44:54,898:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:44:54,908:WARNING: Not Found: /webshow/list/json/orderList!findDyOrderState.action
2018-06-27 18:44:54,913:WARNING: Not Found: /favicon.ico
2018-06-27 18:48:16,224:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:48:16,242:WARNING: Not Found: /favicon.ico
2018-06-27 18:53:43,767:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:53:43,821:WARNING: Not Found: /favicon.ico
2018-06-27 18:53:52,077:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:53:52,096:WARNING: Not Found: /favicon.ico
2018-06-27 18:54:20,075:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:54:20,102:WARNING: Not Found: /favicon.ico
2018-06-27 18:54:21,680:ERROR: Internal Server Error: /webshow/list/ner
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 58, in ner
    demo_sent = list(s.strip())
UnboundLocalError: local variable 's' referenced before assignment
2018-06-27 18:54:22,098:WARNING: Not Found: /favicon.ico
2018-06-27 18:54:29,712:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:54:29,719:WARNING: Not Found: /favicon.ico
2018-06-27 18:54:31,834:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:54:31,846:WARNING: Not Found: /favicon.ico
2018-06-27 18:54:34,271:ERROR: Internal Server Error: /webshow/list/ner
Traceback (most recent call last):
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\exception.py", line 35, in inner
    response = get_response(request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 128, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "C:\Users\Ian\Anaconda3\lib\site-packages\django\core\handlers\base.py", line 126, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "D:\github\web\mysite\webshow\views.py", line 58, in ner
    demo_sent = list(s.strip())
UnboundLocalError: local variable 's' referenced before assignment
2018-06-27 18:54:34,408:WARNING: Not Found: /favicon.ico
2018-06-27 18:56:22,035:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:56:22,042:WARNING: Not Found: /favicon.ico
2018-06-27 18:56:25,529:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:56:25,553:WARNING: Not Found: /favicon.ico
2018-06-27 18:56:28,155:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:56:28,158:WARNING: Not Found: /favicon.ico
2018-06-27 18:56:30,454:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:56:30,505:WARNING: Not Found: /favicon.ico
2018-06-27 18:56:31,959:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-27 18:56:31,972:WARNING: Not Found: /favicon.ico
2018-06-29 09:24:12,125:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-06-29 09:24:28,171:WARNING: Not Found: /favicon.ico
2018-06-29 09:24:36,546:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:24:36,586:WARNING: Not Found: /favicon.ico
2018-06-29 09:25:14,629:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:25:14,632:WARNING: Not Found: /favicon.ico
2018-06-29 09:31:22,564:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:31:22,586:WARNING: Not Found: /favicon.ico
2018-06-29 09:37:41,128:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:37:41,134:WARNING: Not Found: /favicon.ico
2018-06-29 09:47:02,938:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:47:02,949:WARNING: Not Found: /favicon.ico
2018-06-29 09:47:13,397:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:47:13,433:WARNING: Not Found: /favicon.ico
2018-06-29 09:50:34,123:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:50:34,167:WARNING: Not Found: /favicon.ico
2018-06-29 09:53:56,354:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:53:56,368:WARNING: Not Found: /favicon.ico
2018-06-29 09:54:25,912:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:54:25,925:WARNING: Not Found: /favicon.ico
2018-06-29 09:56:48,333:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:56:48,343:WARNING: Not Found: /favicon.ico
2018-06-29 09:57:46,167:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:57:46,179:WARNING: Not Found: /favicon.ico
2018-06-29 09:58:51,635:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:58:51,654:WARNING: Not Found: /favicon.ico
2018-06-29 09:59:28,454:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 09:59:28,470:WARNING: Not Found: /favicon.ico
2018-06-29 10:05:17,690:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 10:05:17,702:WARNING: Not Found: /favicon.ico
2018-06-29 10:06:57,702:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 10:06:57,706:WARNING: Not Found: /favicon.ico
2018-06-29 13:27:37,819:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 13:27:37,831:WARNING: Not Found: /favicon.ico
2018-06-29 15:21:30,844:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:21:30,849:WARNING: Not Found: /favicon.ico
2018-06-29 15:22:12,253:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:22:12,308:WARNING: Not Found: /favicon.ico
2018-06-29 15:23:32,427:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:23:32,444:WARNING: Not Found: /favicon.ico
2018-06-29 15:24:52,610:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:24:52,624:WARNING: Not Found: /favicon.ico
2018-06-29 15:30:39,201:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:30:39,204:WARNING: Not Found: /favicon.ico
2018-06-29 15:33:29,850:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:33:29,861:WARNING: Not Found: /favicon.ico
2018-06-29 15:34:30,947:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 15:34:30,962:WARNING: Not Found: /favicon.ico
2018-06-29 16:51:32,497:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 16:51:32,506:WARNING: Not Found: /favicon.ico
2018-06-29 16:55:54,834:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 16:55:54,849:WARNING: Not Found: /favicon.ico
2018-06-29 16:57:53,623:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 16:57:53,628:WARNING: Not Found: /favicon.ico
2018-06-29 17:02:04,213:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:02:04,242:WARNING: Not Found: /favicon.ico
2018-06-29 17:06:48,565:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:06:48,582:WARNING: Not Found: /favicon.ico
2018-06-29 17:12:50,901:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:12:50,916:WARNING: Not Found: /favicon.ico
2018-06-29 17:15:20,650:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:15:20,663:WARNING: Not Found: /favicon.ico
2018-06-29 17:25:39,684:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:25:39,707:WARNING: Not Found: /favicon.ico
2018-06-29 17:25:57,907:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:25:57,926:WARNING: Not Found: /favicon.ico
2018-06-29 17:33:31,302:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:33:31,316:WARNING: Not Found: /favicon.ico
2018-06-29 17:34:05,112:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:34:05,125:WARNING: Not Found: /favicon.ico
2018-06-29 17:38:00,503:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:38:00,519:WARNING: Not Found: /favicon.ico
2018-06-29 17:40:35,778:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:40:35,795:WARNING: Not Found: /favicon.ico
2018-06-29 17:44:26,038:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:44:26,053:WARNING: Not Found: /favicon.ico
2018-06-29 17:47:50,363:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:47:50,380:WARNING: Not Found: /favicon.ico
2018-06-29 17:49:10,520:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:49:10,534:WARNING: Not Found: /favicon.ico
2018-06-29 17:51:13,585:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:51:13,601:WARNING: Not Found: /favicon.ico
2018-06-29 17:52:07,819:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 17:52:07,834:WARNING: Not Found: /favicon.ico
2018-06-29 18:09:45,064:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 18:09:45,090:WARNING: Not Found: /favicon.ico
2018-06-29 18:11:07,078:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 18:11:07,099:WARNING: Not Found: /favicon.ico
2018-06-29 18:28:14,260:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 18:28:14,274:WARNING: Not Found: /favicon.ico
2018-06-29 18:31:47,032:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 18:31:47,046:WARNING: Not Found: /favicon.ico
2018-06-29 18:48:59,714:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 18:48:59,722:WARNING: Not Found: /favicon.ico
2018-06-29 19:27:58,182:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-06-29 19:27:58,191:WARNING: Not Found: /favicon.ico
2018-07-02 13:07:54,541:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-07-02 13:09:59,603:WARNING: Not Found: /favicon.ico
2018-07-02 13:10:06,004:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 13:10:06,030:WARNING: Not Found: /favicon.ico
2018-07-02 13:10:49,975:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 13:10:49,999:WARNING: Not Found: /favicon.ico
2018-07-02 13:11:05,898:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 13:11:05,911:WARNING: Not Found: /favicon.ico
2018-07-02 13:13:15,049:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 13:13:15,080:WARNING: Not Found: /favicon.ico
2018-07-02 13:14:32,582:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 13:14:32,596:WARNING: Not Found: /favicon.ico
2018-07-02 14:30:40,541:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 14:30:40,558:WARNING: Not Found: /favicon.ico
2018-07-02 14:31:20,945:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-02 14:31:20,957:WARNING: Not Found: /favicon.ico
2018-07-16 13:47:02,311:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-07-16 13:55:29,492:WARNING: Not Found: /favicon.ico
2018-07-16 13:55:50,398:WARNING: Not Found: /webshow
2018-07-16 13:55:50,765:WARNING: Not Found: /favicon.ico
2018-07-16 13:56:42,822:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-16 13:56:43,149:WARNING: Not Found: /favicon.ico
2018-07-18 15:10:54,588:INFO: Restoring parameters from D:\github\web\mysite\webshow/model\1529463214\checkpoints/model-1200
2018-07-18 15:11:11,569:WARNING: Not Found: /favicon.ico
2018-07-18 15:11:21,771:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:11:21,799:WARNING: Not Found: /favicon.ico
2018-07-18 15:11:31,491:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:11:31,509:WARNING: Not Found: /favicon.ico
2018-07-18 15:16:06,841:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:16:06,866:WARNING: Not Found: /favicon.ico
2018-07-18 15:17:19,400:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:17:19,430:WARNING: Not Found: /favicon.ico
2018-07-18 15:17:20,409:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:17:20,438:WARNING: Not Found: /favicon.ico
2018-07-18 15:46:14,736:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:46:14,767:WARNING: Not Found: /favicon.ico
2018-07-18 15:47:41,687:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-07-18 15:47:41,706:WARNING: Not Found: /favicon.ico
2018-08-23 14:19:02,859:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:19:02,971:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:19:04,544:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:19:52,425:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:19:52,537:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:19:53,802:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:21:19,295:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:21:19,406:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:21:20,659:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:21:51,849:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:21:51,959:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:21:53,205:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:22:15,578:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:22:15,691:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:22:17,239:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:30:21,807:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:30:21,918:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:30:23,471:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:30:30,620:ERROR: Invalid HTTP_HOST header: 'h1:11111'. You may need to add 'h1' to ALLOWED_HOSTS.
2018-08-23 14:30:30,818:WARNING: Bad Request: /
2018-08-23 14:30:31,214:ERROR: Invalid HTTP_HOST header: 'h1:11111'. You may need to add 'h1' to ALLOWED_HOSTS.
2018-08-23 14:30:31,249:WARNING: Bad Request: /favicon.ico
2018-08-23 14:32:57,943:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:32:58,052:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:32:59,612:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:34:31,091:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:34:31,201:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:34:32,815:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:34:39,250:ERROR: Invalid HTTP_HOST header: 'h1:11111'. You may need to add 'h1' to ALLOWED_HOSTS.
2018-08-23 14:34:39,301:WARNING: Bad Request: /
2018-08-23 14:34:39,394:ERROR: Invalid HTTP_HOST header: 'h1:11111'. You may need to add 'h1' to ALLOWED_HOSTS.
2018-08-23 14:34:39,429:WARNING: Bad Request: /favicon.ico
2018-08-23 14:34:53,124:WARNING: Not Found: /favicon.ico
2018-08-23 14:38:03,315:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:38:03,425:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:38:04,966:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:38:14,958:WARNING: Not Found: /favicon.ico
2018-08-23 14:38:28,025:ERROR: Invalid HTTP_HOST header: 'h1:11111'. You may need to add 'h1' to ALLOWED_HOSTS.
2018-08-23 14:38:28,075:WARNING: Bad Request: /
2018-08-23 14:38:28,392:ERROR: Invalid HTTP_HOST header: 'h1:11111'. You may need to add 'h1' to ALLOWED_HOSTS.
2018-08-23 14:38:28,430:WARNING: Bad Request: /favicon.ico
2018-08-23 14:41:04,275:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:41:04,389:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:41:05,942:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:41:35,301:WARNING: Not Found: /favicon.ico
2018-08-23 14:44:13,556:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-08-23 14:44:13,585:WARNING: Not Found: /favicon.ico
2018-08-23 14:44:58,367:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-08-23 14:44:58,378:WARNING: Not Found: /favicon.ico
2018-08-23 14:45:02,838:WARNING: Not Found: /webshow/list/json/plan!getItemsInSession.action
2018-08-23 14:45:02,846:WARNING: Not Found: /favicon.ico
2018-08-23 14:45:53,640:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:45:53,755:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:45:55,294:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-23 14:49:23,832:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-23 14:49:23,946:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-23 14:49:25,500:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:38:44,143:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 12:38:44,255:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 12:38:45,807:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:38:45,972:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 12:38:46,009:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:38:46,009:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 12:39:42,032:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:40:26,692:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 12:40:26,801:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 12:40:28,345:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:40:28,499:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 12:40:28,534:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:40:28,534:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 12:41:23,821:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:41:28,472:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 137, in _get_response
    "returned None instead." % (callback.__module__, view_name)
ValueError: The view daozhen.views.getmhzz didn't return an HttpResponse object. It returned None instead.
2018-08-24 12:41:28,898:WARNING: Not Found: /favicon.ico
2018-08-24 12:42:24,415:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 12:42:24,524:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 12:42:26,087:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:42:26,257:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 12:42:26,292:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:42:26,292:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 12:43:23,437:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:43:27,282:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 21, in getmhzz
    return HttpResponse(json.dumps(d, ensure_ascii=False), content_type="application/json")
NameError: name 'HttpResponse' is not defined
2018-08-24 12:43:27,389:WARNING: Not Found: /favicon.ico
2018-08-24 12:45:08,558:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 12:45:08,670:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 12:45:10,281:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:45:10,454:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 12:45:10,490:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:45:10,490:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 12:45:20,630:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 12:45:20,745:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 12:45:22,357:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:45:22,522:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 12:45:22,558:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:45:22,558:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 12:45:41,344:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 12:45:41,452:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 12:45:43,006:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 12:45:43,175:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 12:45:43,210:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:45:43,210:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 12:46:36,862:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 12:46:43,668:WARNING: Not Found: /favicon.ico
2018-08-24 13:26:29,101:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 13:26:29,214:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 13:26:30,782:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 13:26:30,945:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 13:26:30,981:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 13:26:30,981:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 13:27:26,284:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 13:35:58,967:ERROR: Internal Server Error: /daozhen/getmhzz
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/utils/deprecation.py", line 90, in __call__
    response = self.process_request(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/middleware/common.py", line 53, in process_request
    path = self.get_full_path_with_slash(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/middleware/common.py", line 93, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 10.1.192.118:11111/daozhen/getmhzz/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
2018-08-24 13:40:53,086:ERROR: Internal Server Error: /daozhen/getmhzz
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/utils/deprecation.py", line 90, in __call__
    response = self.process_request(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/middleware/common.py", line 53, in process_request
    path = self.get_full_path_with_slash(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/middleware/common.py", line 93, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 10.1.192.118:11111/daozhen/getmhzz/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
2018-08-24 13:44:55,409:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 13:44:55,521:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 13:44:57,068:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 13:44:57,243:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 13:44:57,278:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 13:44:57,278:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 13:45:07,160:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 13:45:07,269:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 13:45:09,019:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 13:45:09,200:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 13:45:09,237:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 13:45:09,238:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 13:46:07,835:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 13:47:07,583:ERROR: Internal Server Error: /daozhen/getmhzz
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/utils/deprecation.py", line 90, in __call__
    response = self.process_request(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/middleware/common.py", line 53, in process_request
    path = self.get_full_path_with_slash(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/middleware/common.py", line 93, in get_full_path_with_slash
    'url': request.get_host() + new_path,
RuntimeError: You called this URL via POST, but the URL doesn't end in a slash and you have APPEND_SLASH set. Django can't redirect to the slash URL while maintaining POST data. Change your form to point to 10.1.192.118:11111/daozhen/getmhzz/ (note the trailing slash), or set APPEND_SLASH=False in your Django settings.
2018-08-24 13:51:44,269:WARNING: Not Found: /favicon.ico
2018-08-24 13:54:49,994:WARNING: Forbidden (CSRF cookie not set.): /daozhen/getmhzz/
2018-08-24 13:57:13,421:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 13:57:13,534:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 13:57:15,157:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 13:57:15,331:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 13:57:15,367:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 13:57:15,367:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 13:58:15,876:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:07:20,071:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:07:20,182:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:07:21,751:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:07:21,913:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:07:21,951:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:07:21,951:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:08:18,932:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:11:08,658:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:11:08,767:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:11:10,335:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:11:10,499:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:11:10,535:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:11:10,535:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:12:06,061:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:12:51,444:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 20, in getmhzz
    print(request.raw_post_data())
AttributeError: 'WSGIRequest' object has no attribute 'raw_post_data'
2018-08-24 14:19:47,888:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:19:47,999:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:19:49,544:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:19:49,714:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:19:49,750:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:19:49,750:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:20:44,084:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:23:59,308:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 20, in getmhzz
    print(request.raw_post_data)
AttributeError: 'WSGIRequest' object has no attribute 'raw_post_data'
2018-08-24 14:24:00,890:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 20, in getmhzz
    print(request.raw_post_data)
AttributeError: 'WSGIRequest' object has no attribute 'raw_post_data'
2018-08-24 14:25:57,944:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:25:58,057:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:25:59,598:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:25:59,769:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:25:59,805:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:25:59,805:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:26:56,170:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:35:50,099:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:35:50,213:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:35:51,761:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:35:51,929:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:35:51,964:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:35:51,965:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:36:45,543:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:44:50,777:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:44:50,887:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:44:52,436:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:44:52,600:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:44:52,636:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:44:52,636:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:45:46,947:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:46:14,081:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 12, in getmhzz
    p = dict(request.body.decode())
ValueError: dictionary update sequence element #0 has length 1; 2 is required
2018-08-24 14:46:41,432:ERROR: Internal Server Error: /daozhen/getmhzz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 12, in getmhzz
    p = dict(request.body.decode())
ValueError: dictionary update sequence element #0 has length 1; 2 is required
2018-08-24 14:49:57,645:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 14:49:57,755:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 14:49:59,308:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 14:49:59,483:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 14:49:59,518:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:49:59,518:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 14:50:55,112:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 14:51:03,430:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 14:51:03,430:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 14:51:03,998:DEBUG: Loading model cost 0.568 seconds.
2018-08-24 14:51:03,998:DEBUG: Prefix dict has been built succesfully.
2018-08-24 15:37:37,278:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:37:37,392:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:37:38,940:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:37:39,112:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:37:39,147:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:37:39,147:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 15:38:35,796:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:41:57,080:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 15:41:57,080:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 15:41:57,670:DEBUG: Loading model cost 0.590 seconds.
2018-08-24 15:41:57,670:DEBUG: Prefix dict has been built succesfully.
2018-08-24 15:46:26,910:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:46:27,023:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:46:28,675:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:46:28,833:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:46:28,869:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:46:28,869:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 15:47:00,176:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:47:00,292:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:47:01,958:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:47:02,136:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:47:02,172:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:47:02,173:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 15:47:44,141:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:47:44,252:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:47:45,870:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:47:46,044:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:47:46,079:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:47:46,080:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 15:48:40,447:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:53:04,939:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:53:05,048:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:53:06,596:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:53:06,761:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:53:06,796:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:53:06,797:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 15:53:31,370:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:53:31,483:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:53:33,110:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:53:33,279:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:53:33,316:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:53:33,316:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 15:54:28,276:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:56:01,291:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 15:56:01,292:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 15:56:01,860:DEBUG: Loading model cost 0.568 seconds.
2018-08-24 15:56:01,860:DEBUG: Prefix dict has been built succesfully.
2018-08-24 15:59:39,381:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 15:59:39,492:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 15:59:41,035:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 15:59:41,208:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 15:59:41,244:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 15:59:41,244:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 16:00:37,044:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:09:19,710:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 16:09:19,710:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 16:09:20,283:DEBUG: Loading model cost 0.573 seconds.
2018-08-24 16:09:20,284:DEBUG: Prefix dict has been built succesfully.
2018-08-24 16:24:28,849:ERROR: Internal Server Error: /daozhen/zz2zz/
Traceback (most recent call last):
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/exception.py", line 34, in inner
    response = get_response(request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 126, in _get_response
    response = self.process_exception_by_middleware(e, request)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/django/core/handlers/base.py", line 124, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "/home/ian/code/github/web/mysite/daozhen/views.py", line 48, in zz2zz
    l1,aa = model.zz2zz(zzlist)
  File "/home/ian/code/github/web/mysite/model/daozhen/model180824.py", line 123, in zz2zz
    zz1 = [i['症状名称'] for i in zzdicts]
  File "/home/ian/code/github/web/mysite/model/daozhen/model180824.py", line 123, in <listcomp>
    zz1 = [i['症状名称'] for i in zzdicts]
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/pymongo/cursor.py", line 1189, in next
    if len(self.__data) or self._refresh():
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/pymongo/cursor.py", line 1104, in _refresh
    self.__send_message(q)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/pymongo/cursor.py", line 982, in __send_message
    helpers._check_command_response(first)
  File "/home/ian/installed/anaconda3/lib/python3.6/site-packages/pymongo/helpers.py", line 155, in _check_command_response
    raise OperationFailure(msg % errmsg, code, response)
pymongo.errors.OperationFailure: $in needs an array
2018-08-24 16:28:36,508:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 16:28:36,621:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 16:28:38,244:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 16:28:38,417:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 16:28:38,453:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:28:38,453:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 16:29:35,646:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:31:09,617:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 16:31:09,618:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 16:31:10,206:DEBUG: Loading model cost 0.589 seconds.
2018-08-24 16:31:10,207:DEBUG: Prefix dict has been built succesfully.
2018-08-24 16:35:02,482:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 16:35:02,592:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 16:35:04,139:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 16:35:04,310:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 16:35:04,346:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:35:04,346:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 16:35:57,882:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:39:32,875:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 16:39:32,875:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 16:39:33,442:DEBUG: Loading model cost 0.567 seconds.
2018-08-24 16:39:33,442:DEBUG: Prefix dict has been built succesfully.
2018-08-24 16:42:53,821:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 16:42:53,935:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 16:42:55,655:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 16:42:55,827:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 16:42:55,863:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:42:55,864:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 16:43:54,183:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:46:41,888:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 16:46:41,888:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 16:46:42,472:DEBUG: Loading model cost 0.584 seconds.
2018-08-24 16:46:42,472:DEBUG: Prefix dict has been built succesfully.
2018-08-24 16:59:11,674:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 16:59:11,783:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 16:59:13,346:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 16:59:13,512:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 16:59:13,547:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 16:59:13,547:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 17:00:09,531:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 17:03:23,786:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 17:03:23,786:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 17:03:24,386:DEBUG: Loading model cost 0.600 seconds.
2018-08-24 17:03:24,387:DEBUG: Prefix dict has been built succesfully.
2018-08-24 17:49:31,625:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/ops/rnn.py:417: calling reverse_sequence (from tensorflow.python.ops.array_ops) with seq_dim is deprecated and will be removed in a future version.
Instructions for updating:
seq_dim is deprecated, use seq_axis instead
2018-08-24 17:49:31,734:WARNING: From /home/ian/installed/anaconda3/lib/python3.6/site-packages/tensorflow/python/util/deprecation.py:432: calling reverse_sequence (from tensorflow.python.ops.array_ops) with batch_dim is deprecated and will be removed in a future version.
Instructions for updating:
batch_dim is deprecated, use batch_axis instead
2018-08-24 17:49:33,420:INFO: Restoring parameters from /home/ian/code/github/web/mysite/webshow/model/1529463214/checkpoints/model-1200
2018-08-24 17:49:33,597:INFO: 'pattern' package not found; tag filters are not available for English
2018-08-24 17:49:33,633:INFO: loading projection weights from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 17:49:33,633:DEBUG: {'kw': {}, 'mode': 'rb', 'uri': '/home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin'}
2018-08-24 17:50:29,711:INFO: loaded (6115353, 64) matrix from /home/ian/code/github/data/word2vec/news_12g_baidubaike_20g_novel_90g_embedding_64.bin
2018-08-24 17:53:57,118:DEBUG: Building prefix dict from the default dictionary ...
2018-08-24 17:53:57,119:DEBUG: Loading model from cache /tmp/jieba.cache
2018-08-24 17:53:57,698:DEBUG: Loading model cost 0.579 seconds.
2018-08-24 17:53:57,699:DEBUG: Prefix dict has been built succesfully.
